app/src/main/java/unipd/se18/ocrcamera/BarcodeRecognizer.java

```diff
@@ -0,0 +1,29 @@
+package unipd.se18.barcodemodule;
+
+/**
+ * Useful set a type of recognition
+ * It avoids the single point of failure relative to this process
+ * @author Andrea Ton
+ */
+public class BarcodeRecognizer {
+
+    /**
+     * Ids of the different barcode api
+     */
```

Comment:
I do not understand the zxing for what it is and what it is for, I would have added a greater description of what it is

Reply:
You are right, zxing is just another api implemented in the original project but not requested for review, so there is its ID on the id's list.

Reply:
I understand, thanks for the explanation